src/EnergyPlus/SetPointManager.cc

```diff
@@ -5422,6 +5757,38 @@ void InitSetPointManagers(EnergyPlusData &state)
                 state.dataSetPointManager->ReturnWaterResetHWSetPtMgr(SetPtMgrNum).maximumHotWaterSetpoint;
         }
 
+        // System Node Reset Temperature and Humidity
+        for (SetPtMgrNum = 1;
```

Comment:
Loop indexes should all be declared at the loop level to keep the scope tight. Don't worry about the existing loops, but for new loops `for (int setPrMgrNum = .....`